### ISSUE TITLE
Funding / PE Campaign and Financial Transparency

### ISSUE THREAD

**Issue by jinaamini** [tcfev/task-force-nika#13]
We need more funding and personnel to work and produce our products ASAP. We should start by campaigning on social media and explaining who we are and what we are working on. Transparency should be a core of funding. 

For financial transparency do we have a protocol?

For basic financial transparency we need to show the public the following:
* How much money have we raised
* How much money have we spent and on what
* Details of our spending as much as possible

**Comment by jinaamini** [comment]
I think we should decouple funding into separate projects and efforts so that you don't have only one option of donating to the organization, and if you want to help in improving internet access in Iran, you can directly donate to that project and we should respect and use that donation only for the donated task in a transparent way.

**Comment by jinaamini** [comment]
This is a suggestion for a financial transparency protocol:

1. One root crypto wallet for the organization
2. Sub-wallets for projects.
3. Metadata should be published after an exchange of crypto to fiat money to provide transparency

This metadata could have redacted parts for security reasons, but as soon as the threat is over we are obliged to publish the unredacted data.

This way, we always show how much money we have raised and how much we have spent. We need to make a similar protocol for old-banking systems to show our capital and expanding.

**Comment by armantorkzaban** [comment]
We can create an intro. doc. on how as an NGO, we are responsible towards the government and money laundry watchdogs of Germany and the EU, and how our interaorganisational principles, and routines guarantee transparency in income and our expenditure to be in line with the [goals of our organisation](https://github.com/tcfev/task-force-nika/blob/main/assets/TCFeV-fields-of-work.pdf). 
 
Externally, we are responsible towards the regulations of the Finanzamt of the state of Niedersachsen of Germany. They watch over our shoulders to make sure we keep it clean. We report yearly to them.
Aside from that there is an EU-wide money laundry watch dogs that assures we are not doing crazy stuff, by analysing our bank transactions.
Next to all the above there the regulations of the corporations we receive services from (Paypal, ...). These also watch carefully what we do.

Then we have our financial transparency protocols within the organisation: 
1. first we have this rule of three which makes sure that at least three people are informed about each action taken.
2. or the principle that the decisions about expenditures exceeding the upper limit of 200€ (for now) should be made by the board with the consent of a relative majority of the members.
---
We two have talked about the role of financial transparency officer to be added to the organisation. I have talked about that with @marlenefuerdem as well, and she is on board.
We need to inform @prettyvulture about that and then we can take it to the GA and get the consent of other members; I am sure they will love and accept it as well.
 
Maybe you write a bit about the specification of the role here so we can have it fully documented prior to the meeting. WDYT?

**Comment by armantorkzaban** [comment]
Let us separate Public Engagement from this issue and make it a super-issue of this one. WDYT @jinaamini?

**Comment by jinaamini** [comment]
> Let us separate Public Engagement from this issue and make it a super-issue of this one. WDYT @jinaamini?

Yes!

> We can create an intro. doc. on how as an NGO, we are responsible towards the government and money laundry watchdogs of Germany and the EU, and how our interaorganisational principles, and routines guarantee transparency in income and our expenditure to be in line with the [goals of our organisation](https://github.com/tcfev/task-force-nika/blob/main/assets/TCFeV-fields-of-work.pdf).
> 
> Externally, we are responsible towards the regulations of the Finanzamt of the state of Niedersachsen of Germany. They watch over our shoulders to make sure we keep it clean. We report yearly to them. Aside from that there is an EU-wide money laundry watch dogs that assures we are not doing crazy stuff, by analysing our bank transactions. Next to all the above there the regulations of the corporations we receive services from (Paypal, ...). These also watch carefully what we do.
> 
> Then we have our financial transparency protocols within the organisation:
> 
>     1. first we have this rule of three which makes sure that at least three people are informed about each action taken.
> 
>     2. or the principle that the decisions about expenditures exceeding the upper limit of 200€ (for now) should be made by the board with the consent of a relative majority of the members.
> 
> 
> We two have talked about the role of financial transparency officer to be added to the organisation. I have talked about that with @MarleneFuerDem as well, and she is on board. We need to inform @prettyvulture about that and then we can take it to the GA and get the consent of other members; I am sure they will love and accept it as well.
> 
> Maybe you write a bit about the specification of the role here so we can have it fully documented prior to the meeting. WDYT?

Great! I will make a proposal for the roles.

We should ask for bank xml/json API otherwise this would not scale. Do we have wallet software internally?  


Financial transparency feature list:
* capital of the NPO at any moment
* history of transactions
* categorization by project and type and other metadata 
* senders identity can be private or public upon consent. 
* after the transaction, an tracking id is given to the user
* tracking id can track into wich project the money has been gone
*  statistics are displayed with graphs and charts and the best visual aids for a good user experience
* metadata should be transparent as possible 
* metadata follows our protective transparency code ( to be defined)
  *backend APIs have open access.

**Comment by MarleneFuerDem** [comment]
Hello there, I like the push for more transparency. @jinaamini should we meet and talk about it? @armantorkzaban did you take over finishing the excel file or should I do that the next days? One thought: to be realistic, the 200 € boundary and that we always have to call for a general meeting is not practicable. maybe we can increase that to 1000 or 2000 €? WDYT? Maybe a topic for the next meeting?

**Comment by armantorkzaban** [comment]
Hi  @MarleneFuerDem, 
I added a couple of things to the whole.
- 2 payments done over Remitely. (TCF account)
- All the payments that I had made personally to people and places where TCF could not send (refugees in Iraq & Syria - sanctioned countries), in a separate files. 
- commented on an entry marked yellow and explained
- tagged you in the second list and asked for assistance on a matter. 

@prettyvulture is checking my entries and I see that he is almost 80% through the list.
I would like to ask you to take a look and confirm nothing is missing. 

>One thought: to be realistic, the 200 € boundary and that we always have to call for a general meeting is not practical. maybe we can increase that to 1000 or 2000 €? WDYT? Maybe a topic for the next meeting?

I would suggest that we clarify in the bylaws the following:

- GA decides whether or not delegates/speakers have the mandate 
- Once given the mandate, each three delegates/speaker can make decision on expenditures of up to xxxx$ ( 10K?)
- Members will be notified about details of expenditure on a regular basis
- To alter the limit any member can call for GA in a normal call procedure

**Comment by lukas-h** [comment]
Hi everyone,

I think it would be good from an OpSec standpoint to not share
- modes of payment (payment services, crypto currencies)
- exact payment amounts
- wallet addresses

It opens new attack surfaces.

I think these discussions should be moved to one of the secure messengers for TCF members only. 

For the public a monthly report of income and expenses should be sufficient imo.

**Comment by jinaamini** [comment]
A traceability function to see where your donation went is essential for financial transparency. The protocol should protect privacy of the users as well. Details such as bank IDs are of course redacted, but I believe the amount should of the donation should be present.

**Comment by armantorkzaban** [comment]
Thank you all for your contributions to this important discussion.
Should we already start compiling the protocol's text together?

**Comment by MarleneFuerDem** [comment]
yes, we need an extensive protocol on the one hand and changes in the constitution. For the constitution I agree with Arman and suggest the following: 

The general meeting decides whether or not members of the board have the mandate to spend the associations funds.
Once given the mandate, each member of the board can make decision on expenditures of up to  10 000 €.
Members will be notified about details of the expenditure on a monthly basis. Members can ask the board for further clarifications if questions arise. 
In order to alter the limit on expenditures any member can call for general meeting according to the procedure explained in § 8. 

If you agree, we can include that in the meeting on Saturday and vote on it.

**Comment by osAIran** [comment]
For radical transparnecy we can add the member's privacy-protected information. This info should include CV, About Me, Goals, Work.

To have a transparency protocol we need two things:

1. Open Data API, Open Source
2. Systematic design of a user experience for the objective of transparency.

For the first task, I can start working as soon as I have the data. Our code sources are mostly open source, but without any explainability to the users. I think we need more resources here and a systematic approach to transparency. We can use design sprints to brainstorm, create value proposition canvases, create user profiles, and journey maps, what users want, how they interact with the system, etc. Design sprints need a team of about 5 and a team lead. To have a greater impact, we should do this with more resources as this could potentially scale high, and a bad impression is hard to recover.

**Comment by osAIran** [comment]
I've created a [repository](https://github.com/tcfev/financial-transparency
) to start this initiative. Upon TCF's approval, I can start first by uploading the raw document that TCF gave.

We can also publicly share our financial google to social media and ask others to supervise our finance.
The document is in a google doc format and I have yet to work with google APIs to read docs and parse the table into JSON/XML or other digital formats. If there is an expert in this I'd appreciate your help.

We need to design a sustainable financial transparency protocol. Manual work will not scale and we need automation scripts and APIs. For now, it is a great initiative to start with the data we have, and in the future, it should become automated cron jobs.

**Comment by armantorkzaban** [comment]
Please go ahead and share the anonymized version on the repo. you have created. We can then link to it from other sources i.e transcf.org, twitter.com/transcf org. GH's readme, etc.
